tools/Az.Tools.Predictor/Az.Tools.Predictor/command_param_to_resource_map.json

```diff
@@ -1 +1 @@
eid": "moduleid", "resourcegroupname": "resourcegroupname", "resourceid": "resourceid"}, "iothubdevicemethod": {"deviceid": "deviceid", "iothubname": "iothubname", "name": "iothubdevicemethodname", "payload": "payload", "resourcegroupname": "resourcegroupname", "resourceid": "resourceid"}, "applyupdate": {"applyupdatename": "applyupdatename", "providername": "providername", "resourcegroupname": "resourcegroupname", "resourcename": "resourcename", "resourceparentname": "resourceparentname", "resourceparenttype": "resourceparenttype", "resourcetype": "resourcetype"}, "configurationassignment": {"providername": "providername", "resourcegroupname": "resourcegroupname", "resourcename": "resourcename", "resourceparentname": "resourceparentname", "resourceparenttype": "resourceparenttype", "resourcetype": "resourcetype", "configurationassignmentname": "configurationassignmentname", "location": "location", "maintenanceconfigurationid": "maintenanceconfigurationid", "resourceid": "resourceid"}, "maintenanceconfiguration": {"name": "maintenanceconfigurationname", "resourcegroupname": "resourcegroupname", "location": "location", "maintenancescope": "maintenancescope", "expirationdatetime": "expirationdatetime", "recurevery": "recurevery", "startdatetime": "startdatetime", "timezone": "timezone", "visibility": "visibility"}, "maintenanceupdate": {"providername": "providername", "resourcegroupname": "resourcegroupname", "resourcename": "resourcename", "resourceparentname": "resourceparentname", "resourceparenttype": "resourceparenttype", "resourcetype": "resourcetype"}, "insightsprivatelinkscope": {"name": "insightsprivatelinkscopename", "resourcegroupname": "resourcegroupname", "resourceid": "resourceid", "location": "location"}, "insightsprivatelinkscopedresource": {"name": "insightsprivatelinkscopedresourcename", "resourcegroupname": "resourcegroupname", "resourceid": "resourceid", "scopename": "scopename", "linkedresourceid": "linkedresourceid"}, "p2svpngatewayvpnconnection": {"resourceid": "resourceid"}, "virtualnetworkgatewayvpnconnection": {"resourcegroupname": "resourcegroupname", "resourcename": "virtualnetworkgatewayname", "virtualnetworkgatewayname": "virtualnetworkgatewayname", "resourceid": "resourceid"}, "privatelinkresource": {"privatelinkresourceid": "privatelinkresourceid", "privatelinkresourcetype": "privatelinkresourcetype", "resourcegroupname": "resourcegroupname", "servicename": "servicename"}, "applicationgatewayfirewallpolicyexclusion": {"matchvariable": "matchvariable", "selector": "selector", "selectormatchoperator": "selectormatchoperator"}, "applicationgatewayfirewallpolicymanagedrulegroupoverride": {"rulegroupname": "rulegroupname"}, "applicationgatewayfirewallpolicymanagedruleoverride": {"ruleid": "ruleid", "state": "state"}, "applicationgatewayfirewallpolicymanagedruleset": {"rulesettype": "rulesettype", "rulesetversion": "rulesetversion"}, "applicationgatewayfirewallpolicysetting": {"mode": "mode", "state": "state"}, "applicatio
```

Comment:
not sure why this file was touched, can you please restore this change in your PR?

Reply:
Thanks for reviewing! I reverted the change.